src/main/java/com/google/step/model/Deal.java

```diff
@@ -18,7 +19,8 @@ public Deal(
       String end,
       String source,
       long posterId,
-      long restaurantId) {
+      long restaurantId,
+      String timestamp) {
```

Comment:
nit: Find a better name. What kind of timestamp is it? `creationTimestamp` / `lastUpdateTimestamp` / `dealStartTimestamp`?